src/parser/ad_parser.js

```diff
@@ -326,10 +329,7 @@ export function _parseAdVerifications(verifications) {
     const verification = new AdVerification();
     const childNodes = verificationNode.childNodes;
 
-    parserUtils.assignAttributes(
-        verificationNode.attributes,
-        verification
-    );
+    parserUtils.assignAttributes(verificationNode.attributes, verification);
```

Comment:
is this needed if you're calling it in line 42?

Reply:
It's not part of my code, Prettier did that 😅